CsvParser/csvParser.go

```diff
@@ -85,4 +86,47 @@ func CsvParser(filePath string) [100]incident.DefaultIncident{
 	return incidentArray
 }
 
- 
\ No newline at end of file
+func convertDefaultIncidentToIncident(arr []incident.DefaultIncident) []incident.Incident {
+	incidentArr := make([]incident.Incident, len(arr), len(arr))
+	for i := range arr {
+		incidentArr[i] = &arr[i]
+	}
+	return incidentArr
+}
+
+func makeJsonObjFile(arr []incident.Incident) *os.File {
+	// numObjects determines how many incidents are stored in the json
+	const numObjects = 1
+	f, err := os.Create("incidents.json")
+	var objs [numObjects]incident.IncidentData
+	if err != nil {
+		fmt.Println(err)
+		return f
+	}
+
+	for i := 0; i < numObjects; i++ {
+		gpStart, gpEnd := arr[i].GetGoodPeriod()
+		bpStart, bpEnd := arr[i].GetBadPeriod()
+		gMetric := arr[i].GetGoodMetric()
+		bMetric := arr[i].GetBadMetric()
+		severity := arr[i].GetSeverity()
+		testsAffected := arr[i].GetTestsAffected()
+		gpInfo := arr[i].GetGoodPeriodInfo()
+		bpInfo := arr[i].GetBadPeriodInfo()
+		iInfo := arr[i].GetIncidentInfo()
```

Comment:
may be space out these into like 3 three chunks but it is not that important at all